Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "non-fungible-token-wrapper"
+version = "0.0.2"
+authors = ["Near Inc <hello@nearprotocol.com>"]
+edition = "2018"
+
+[dev-dependencies]
+near-sdk = "3.1.0"
```

Comment:
I believe in our examples we've been putting this under `[dependencies]` instead of `[dev-dependencies]` like this:
https://github.com/near/near-sdk-rs/blob/master/examples/non-fungible-token/nft/Cargo.toml#L10-L11

Reply:
Good catch. I didn't notice this at all